kernal/drivers/x16/screen.s

```diff
@@ -808,6 +816,58 @@ cpylaeB:	lda #$d8
 	ldx #3
 	jmp copyv
 
+cpykatC:
+	lda #$d8
+	sta tmp2+1       ;character data at ROM 1800
+	ldx #5
+	jsr copyv
+	lda #$f4
+	sta tmp2+1       ;character data at ROM 3400
+	ldx #3
+	jsr copyv
+
+	; copying individual characters
+
+@vram_backslash	=$f000+($5c*8)
+@chariso2_yen	=$d800+($a5*8)
+@vram_tilde 	=$f000+($7e*8)
+@charkat_tilde	=$f400+($00*8)
+@vram_nbsp  	=$f000+($a0*8)
```

Comment:
For the ROM locations of the glyphs, I would rather avoid using constants like this.  cc65 has a nice way of specifying the offset from the beginning of a segment.

Inside the charset-x16.cfgtpl, in the SEGMENTS section, add `define = yes` to each of the segment definitions, and you can then import the symbol for them in this file and define them here.

For instance
`.import __CHARISO2_LOAD__`

`@chariso2_yen = __CHARISO2_LOAD__+(a5*8)`

For the VRAM addresses, we are already doing `.include "io.inc"` in this file, so you can use the constant `charset_addr`

For instance,
`@vram_nbsp = charset_addr+(a0*8)`

Reply:
I like that!

Reply:
I'll deal with that tomorrow

Reply:
@mooinglemur compiler has trouble figuring out the segment stuff